spec/features/tag_layout_template_spec.rb

```diff
@@ -20,14 +20,14 @@
     expect(page).to have_content 'Tag Layout Template New'
     within('#new_tag_layout_template') do
       fill_in('tag_layout_template_name', with: 'Test tag layout template')
-      select('InColumns', from: 'tag_layout_template_direction_algorithm')
```

Comment:
Retry InColumns in case Eduardo's change fixed whatever the problem was.

Reply:
Checking in another branch (testing_feature_tag_layout_in_columns)

Reply:
Reverted, was due to ambiguity due to new InColumnsThenColumns also matching, used InColumns (A1,B1,C1...) instead.